python/jupytergis_qgis/jupytergis_qgis/qgis_loader.py

```diff
@@ -151,13 +157,12 @@ def qgis_layer_to_jgis(
         source_type = "GeoJSONSource"
         source = layer.source()
 
-        components = source.split("/")
-
-        # Get the last component, which should be the file name
-        file_name = components[-1]
-
-        # Remove any query parameters
-        file_name = file_name.split("|")[0]
+        if source.startswith("http://") or source.startswith("https://"):
+            file_name = source
+        else:
+            components = source.split("/")
+            file_name = components[-1]
+            file_name = file_name.split("|")[0]
```

Comment:
Does this actually work when the local file is not in the same folder as the JGIS file?

e.g. if the JGIS file points to the geojson file in another folder `../path/to/my.geo.json`?

I understand from the code only `my.geo.json` will be kept

Reply:
Let's track this in an issue!